### PR TITLE
OKTA-365443: schema changes for optional details property

### PR DESCRIPTION
**Pull request by vikrammoharir-okta**
OKTA-365443: add optional `details` property on schema validation

### CODE REVIEW

File: scripts/workflow-schema.json [hunk 1]
```diff
@@ -39,6 +39,23 @@
                 "title": "The first anyOf schema"
             }
         },
+        "details": {
+            "$id": "#/properties/details",
+            "type": "object",
+            "title": "The details schema",
+            "required": [
```
Comment (charleschan-okta) [inline scripts/workflow-schema.json:46]: do we want to make this required? (Richard's comment in the Jira is to make it optional for now, and for next months MC change to mandatory)

**Review by charleschan-okta — APPROVED** [review]
Pl confirm if want to make change optional or mandatory, otherwise good to go

File: scripts/workflow-schema.json (hunk 1 above)
Comment (vikrammoharir-okta) [reply scripts/workflow-schema.json:46]: so this `required` field is only applicable when `details` section exists. e.g if their is no details than L46 wont be executed. 
If we want to make `details` mandatory, then those changes goes to its parent element i.e [here](https://github.com/okta/workflows-templates/pull/59/files#diff-ce9136e9a6addece2072cec7cf518722e54bb7a7bf3acb6055a671695dccc467L7)

Comment (charleschan-okta) [reply scripts/workflow-schema.json:46]: thanks for clarifying

**Pull request merged**